src/detect-engine-alert.c

```diff
@@ -416,6 +416,7 @@ void PacketAlertFinalize(DetectEngineCtx *de_ctx, DetectEngineThreadCtx *det_ctx
 
             /* pass "alert" found, we're done */
             if (pa->action & ACTION_PASS) {
+                p->action |= ACTION_PASS;
```

Comment:
hmm we had done quite a nice cleanup before, removing the flag from the `Packet::action`. Since it is only for logging, I wonder if we can keep it out of this field to make clear it's not used otherwise.

Reply:
> hmm we had done quite a nice cleanup before, removing the flag from the `Packet::action`. Since it is only for logging, I wonder if we can keep it out of this field to make clear it's not used otherwise.

Would it make sense to have a field for the verdict itself? I can try to access the last packet alert in the queue, which should be the one with the pass rule... 🤔